content/docs/operating/configuration.md

```diff
@@ -399,7 +399,7 @@ Prometheus relabeling to control which instances will actually be scraped. Also
 by default all apps will show up as a single job in Prometheus (the one specified
 in the configuration file), which can also be changed using relabeling.
 
-### Zookeeper Serverset SD configurations `<serverset_sd_config>`
+### `<serverset_sd_config>`
```

Comment:
Ah, GitHub's collapsed code view played a trick on me, sorry.

Yeah, I think it's fine to remove the angled brackets.